test/model/autocomplete_test.dart

```diff
@@ -327,6 +327,11 @@ void main() {
     doCheck('full full', eg.user(fullName: 'Full Full Name'), true);
     doCheck('full full', eg.user(fullName: 'Full Name Full'), true);
 
+    // assert that name checking is diacritic insensitive
```

Comment:
This could be enhanced with test groups but that will be against the pattern in this file, I imagine some little tweaks would be required.
I would prefer updating it but I try to follow the patterns I find in the codebase.
So.. what do you suggest?